app/src/main/java/com/iron/espresso/presentation/home/mystudy/MyStudyDetailActivity.kt

```diff
@@ -92,8 +97,10 @@ class MyStudyDetailActivity :
 
     override fun onPrepareOptionsMenu(menu: Menu?): Boolean {
         if (authority == AUTH_HOST) {
-            val item = menu?.findItem(R.id.delete_study)
-            item?.isVisible = true
+            val deleteItem = menu?.findItem(R.id.delete_study)
+            val delegateItem = menu?.findItem(R.id.host_delegate)
+            deleteItem?.isVisible = true
+            delegateItem?.isVisible = true
```

Comment:
```suggestion
            menu?.findItem(R.id.delete_study)?.isVisible = true
            menu?.findItem(R.id.host_delegate)?.isVisible  = true
```
local 변수 에 대해 더이상 하는 작업이 없고 추가 작업이 생길 가능성이 없어보이면, 이렇게 처리해도 괜찮을 것 같아요

Reply:
저도 코딩하면서 가독성 때문에 살짝 애매했는데 이 정돈 괜찮아 보입니다! 수정했습니다!